go.mod

```diff
@@ -43,16 +43,16 @@ replace (
 	github.com/evanphx/json-patch => github.com/evanphx/json-patch v0.0.0-20191220184958-bf22ed931162
 	github.com/fsnotify/fsnotify => github.com/fsnotify/fsnotify v1.4.7
 	github.com/gin-contrib/sse => github.com/gin-contrib/sse v0.0.0-20170109093832-22d885f9ecc7
-	github.com/gin-gonic/gin => github.com/gin-gonic/gin v0.0.0-20180501062418-bd4f73af679e
+	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.2
 	github.com/go-gl/glfw/v3.3/glfw => github.com/go-gl/glfw/v3.3/glfw v0.0.0-20200222043503-6f7a984d4dc4
 	github.com/go-logr/logr => github.com/go-logr/logr v0.4.0
 	github.com/go-openapi/jsonpointer => github.com/go-openapi/jsonpointer v0.19.3
 	github.com/go-openapi/jsonreference => github.com/go-openapi/jsonreference v0.19.3
 	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
 	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
 
-	github.com/go-playground/assert/v2 => github.com/go-playground/assert/v2 v2.0.0-20160126235308-4f4dfbc7d1c4
-	github.com/go-playground/universal-translator => github.com/go-playground/universal-translator v0.0.0-20160126235308-2223ab324566
+	github.com/go-playground/assert/v2 => github.com/go-playground/assert/v2 v2.0.1
+	github.com/go-playground/universal-translator => github.com/go-playground/universal-translator v0.17.0
```

Comment:
DEPENDENCIES.md has `github.com/go-playground/universal-translator@v0.18.0`

Reply:
go.mod files are kind of a mess -- the only way I know to get all (incl. transitive) dependencies is `go mod graph`. 

I _think_ what what's happening currently is that `go.mod`

* `require`s v0.18.0
* `replace`s all versions of that dependency with v0.17.0

Reply:
might have fixed this now, please re-review?